### PR TITLE
Integrate CC test case 'audit-remote' into openQA

### PR DESCRIPTION
**Pull request by Amrysliu**
- Related ticket: https://progress.opensuse.org/issues/96735
- Needles: N/A
- Verification run: 
- https://openqa.suse.de/tests/8039923#  (s390x)
- https://openqa.suse.de/tests/8039398# (x86_64)
- https://openqa.suse.de/tests/8039657#  (aarch64)

### CODE REVIEW

File: schedule/security/cc_audit_remote.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: cc_ipsec
+description:    >
+    This is for ipsec test in CC system role
```
Comment (rfan1) [inline schedule/security/cc_audit_remote.yaml:3]: ipsec ->  audit remote

File: tests/security/cc/audit_remote.pm [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # Export SYSTEMD_PAGER to let the journalctl exits automatically
+    assert_script_run('export SYSTEMD_PAGER=""');
+
+    if ($test_node eq 'server') {
+        my $pid = background_script_run("$audit_test::test_dir/audit-test/utils/network-server/lblnet_tst_server");
+
+        mutex_create('AUDIT_REMOTE_SERVER_READY');
+        wait_for_children;
+
+        assert_script_run("ip addr del $server_ip/24 dev $netdev") if (is_s390x);
```
Comment (rfan1) [inline tests/security/cc/audit_remote.pm:49]: How about add a comment here?

File: schedule/security/cc_audit_remote.yaml (hunk 1 above)
Comment (Amrysliu) [reply schedule/security/cc_audit_remote.yaml]: Thanks, fixed

File: tests/security/cc/audit_remote.pm (hunk 2 above)
Comment (Amrysliu) [reply tests/security/cc/audit_remote.pm:50]: Added

File: tests/security/cc/audit_remote.pm [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run CC 'audit-remote test case'
```
Comment (jouyingbin) [inline tests/security/cc/audit_remote.pm:6]: Does it can be,
Run CC **'audit-remote'** test case ?

**Review by rfan1 — APPROVED** [review]
lgtm

**Review by lilyeyes — APPROVED** [review]
LGTM

File: tests/security/cc/audit_remote.pm (hunk 3 above)
Comment (Amrysliu) [reply tests/security/cc/audit_remote.pm]: Fixed. Thx

**Review by jouyingbin — APPROVED** [review]
LGTM

**Pull request merged**